src/swig/Makefile

```diff
@@ -6,10 +6,10 @@ all clean:
 	for subdir in $$list; do \
 		if [ -x $$subdir/build -a ! -f .$$subdir -o $@ = clean ] ; \
 		then echo -n Building $$subdir... ; \
-			cd $$subdir && output=$$(CXXFLAGS="$(CXXFLAGS)" ./build $@ 2>&1) ; \
```

Comment:
Great contribution! But why this change?

Reply:
Putting output into a variable and echoing the variable works weird in my mac. Line-breaks are missing.

![compare](https://user-images.githubusercontent.com/1331413/44828982-fc964a80-ac4c-11e8-8276-2a809a794c89.jpg)

And I believe that build shell's output which might be very important and helpful for those who build their own libraries to find out which step goes wrong or warning shall be always shown.

Reply:
Same on Linux, and I noticed some useful warnings (e.g. operator=) while testing this change even when successful.